test/README.md

```diff
@@ -30,6 +30,7 @@ in local kubeconfig file (~/.kube/config by default) in you local machine.
 is the difference of running environments. The envs that our presubmit test
 uses are stored in ./*.env files. Specifically,
 > - e2e-tests-kind-prow-alpha.env for [`pull-tekton-pipeline-alpha-integration-tests`](https://github.com/tektoncd/plumbing/blob/d2c8ccb63d02c6e72c62def788af32d63ff1981a/prow/config.yaml#L1304)
+> - e2e-tests-kind-prow-beta.env for [`pull-tekton-pipeline-beta-integration-tests`](https://github.com/tektoncd/plumbing/blob/d2c8ccb63d02c6e72c62def788af32d63ff1981a/prow/config.yaml#L1357)
```

Comment:
It seems like the link does not match the text 🤔

Reply:
Thanks for the catch, will change to permanent link

Reply:
Actually for this it seems we have to wait for https://github.com/tektoncd/plumbing/pull/1256

Reply:
Ahh, either we need to remove this in the PR and create a TODO issue to catch it as a followup, or we need to stop using permanent link here, or we hold this PR untile that PR gets merged.

wdyt?

Reply:
Thanks, updated